<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-AT2K5M2" failures="1" tests="2" name="Default test" time="65.168" errors="0" timestamp="2021-07-05T12:35:58 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="30.876"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="33.389">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at stepDefinations.StepDefinations.filter_by_and_check_projects_count_is(StepDefinations.java:27)
at âœ½.filter by "QA" and check projects count is 3(file:///C:/Users/vigne/Documents/EclipseWorkspace/Automation/Fleet_Studio/src/test/resources/Features/Workpage.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
